linkedin/utils.py

```diff
@@ -16,6 +13,22 @@
         import json
 
 
+if sys.version < '3':
```

Comment:
Isn't it better to use ``sys.version_info < (3,)``? (Not that Python 10 is on the horizon)